### PR TITLE
[Merged by Bors] - feat: port Data.Pfunctor.Univariate.M

### PR DESCRIPTION
**Pull request by alexkeizer**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/PFunctor/Univariate/M.lean [hunk 1]
```diff
@@ -0,0 +1,757 @@
...
+    apply h₂
+#align pfunctor.M.bisim PFunctor.M.bisim
+
+theorem bisim' {α : Type _} (Q : α → Prop) (u v : α → M P)
+    (h :
+      ∀ x,
+        Q x →
+          ∃ a f f',
+            M.dest (u x) = ⟨a, f⟩ ∧
+              M.dest (v x) = ⟨a, f'⟩ ∧ ∀ i, ∃ x', Q x' ∧ f i = u x' ∧ f' i = v x') :
```
Comment (jcommelin) [inline Mathlib/Data/PFunctor/Univariate/M.lean:701]: Please reflow this.

**Review by jcommelin — COMMENTED** [review]
Thanks!

bors d+

**Comment by alexkeizer** [comment]
@jcommelin I've had to add 
```lean
set_option linter.uppercaseLean3 false in
```
to a bunch of aligns to make it build on the latest master.

Is it still OK to merge without another round of review?

**Pull request closed without merging**